android/storage.md

```diff
@@ -32,13 +32,15 @@ See [Authentication](./authentication) for more information on how to get the `u
     $ cd ./YOUR_PROJECT_FOLDER
     $ amplify add storage
     ```
-3.  Choose `Content` as your storage service.
+3. Choose `Content` as your storage service.
 
     `❯ Content (Images, audio, video, etc.)`
 
-4. The CLI walks you through the options to enable Auth (if not enabled previously), to name your S3 bucket, and to decide who should have access (select `Auth and guest users` and toggle all to select `create/update, read, and delete` access for both auth and guest users).
+4. The combination of friendly name and bucket name must be globally unique. If another S3 user has specified the same values for both of these as you, the amplify push step below will fail.
 
-5. Confirm that you have storage and auth set up.
+5. The CLI walks you through the options to enable Auth (if not enabled previously), to name your S3 bucket, and to decide who should have access (select `Auth and guest users` and toggle all to select `create/update, read, and delete` access for both auth and guest users).
+
+6. Confirm that you have storage and auth set up.
```

Comment:
nit: I think our style is to capitalize category names

Reply:
Done